catalogd/internal/storage/index.go

```diff
@@ -50,20 +50,6 @@ func (s *section) UnmarshalJSON(b []byte) error {
 	return nil
 }
 
-func (i index) Size() int64 {
```

Comment:
why are we removing this one?

Reply:
Good question. I realized it's not being used anywhere, so at least in the context of existing code, it's dead code.

Reply:
Yep, I was using that in my prototype to estimate how big the index would be in memory. No need for it anymore.